new pull

alexander01202 opened the pull request:
"Destination completed"

andrew-bierman reviewed (approved):
looks good, only concern is about the server index.ts change ?

alexander01202 commented:
> looks good, only concern is about the server index.ts change ?

Nothing too major done here. The express app only starts listening after a successful connection to the database. Helps to understand where the problem is coming from in case of any failed connection. 

_Can be ignored if considered not necessary_

andrew-bierman commented:
Crashes on destination route. 

- [ ] Move hooks on /components/destination out of conditional now 
- [ ] We shouldn't be using the photonDetailsStore now that we are getting data from query hooks

andrew-bierman commented:
From the dashboard screen, are you able to search for a destination and then click on an option and see the info on that destination page? I'm not able to on either on my environments

alexander01202 commented:
I am not able to either. That's why I was asking you what was meant to be passed on discord. The error doesn't seem to come from my implementation. The query is just failing. Probably something from the backend. I have tried passing in `destinationId` which is `query` and also `Id` which is just a number but it still fails. 

If `destinationId` is not meant to return `query` when logged into the console, then perhaps that's the problem. Maybe the search parameters are wrong.

What do you think?

andrew-bierman commented:
> I am not able to either. That's why I was asking you what was meant to be passed on discord. The error doesn't seem to come from my implementation. The query is just failing. Probably something from the backend. I have tried passing in `destinationId` which is `query` and also `Id` which is just a number but it still fails.
> 
> If `destinationId` is not meant to return `query` when logged into the console, then perhaps that's the problem. Maybe the search parameters are wrong.
> 
> What do you think?

It works fine on andrew_testing, it must be something in our destination page refactor i think. now that we are not using the store we aren't handling the response properly. the destinationId being query is fine, its that way on andrew_testing branch.

alexander01202 commented:
Alright would get working on it

alexander01202 commented:
Destination should work now. Confirmed from my end. Please verify

andrew-bierman commented:
Ok yes its working but it seems like we just reverted back to using the old logic combined with the store ?

alexander01202 commented:
No andrew!! 😂 I didn’t do that. All the previous store imports are not
used anywhere. I can explain how I fixed that if you need me to. I won’t
mind doing that . Should I proceed?

On Wed, Nov 22, 2023 at 14:36 Andrew Bierman ***@***.***>
wrote:

> Ok yes its working but it seems like we just reverted back to using the
> old logic combined with the store ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/andrew-bierman/PackRat/pull/488#issuecomment-1822784451>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUJELUMUGNZVRLVMZ3CZPQLYFX5ULAVCNFSM6AAAAAA7HQYE3GVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQMRSG44DINBVGE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

andrew-bierman commented on client/components/destination/index.js:
```diff
@@ -109,13 +110,11 @@ const WeatherData = ({ geoJSON }) => {
-            // const weatherWkRes = await dispatch(fetchWeatherWeek({ lat, lon }));
-            // dispatch(setWeatherObject(weatherObjRes.payload));
-            // dispatch(setWeatherWeek(weatherWkRes.payload.list.slice(0, 4)));
+            const weatherObjRes = await dispatch(fetchWeather({ lat, lon }));
```
This and the change above is what i was referring to

andrew-bierman replied:
Yes that's correct. I'm just referring to the files changed in this PR, it looks like there are modifications to the weather logic. I know that this PR is just for destination. 

If you look at the diff in files changed on GitHub, that's what I'm referring to.

I think it was unintentional change just due to force push previously, branches are likely out of sync. I can revert just wanted to confirm if that change was intended to be included in this PR. All good.

alexander01202 replied:
Yeah. I didn't make those changes.

andrew-bierman commented:
> No andrew!! 😂 I didn’t do that. All the previous store imports are not used anywhere. I can explain how I fixed that if you need me to. I won’t mind doing that . Should I proceed?
> […](#)
> On Wed, Nov 22, 2023 at 14:36 Andrew Bierman ***@***.***> wrote: Ok yes its working but it seems like we just reverted back to using the old logic combined with the store ? — Reply to this email directly, view it on GitHub <[#488 (comment)](https://github.com/andrew-bierman/PackRat/pull/488#issuecomment-1822784451)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AUJELUMUGNZVRLVMZ3CZPQLYFX5ULAVCNFSM6AAAAAA7HQYE3GVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQMRSG44DINBVGE> . You are receiving this because you authored the thread.Message ID: ***@***.***>

Ok I might be misunderstanding the changes to the useFetchWeather logic then, my apologies

alexander01202 commented:
You're not wrong Andrew. I wasn't tasked with weather store.
